src/xunit.console/CommandLine.cs

```diff
@@ -395,6 +396,71 @@ protected XunitProject Parse(Predicate<string> fileExists)
             return project;
         }
 
+        static void IncludeFromFile(XunitProject project, String path)
+        {
+            using (var input = File.OpenText(path))
```

Comment:
Consider splitting the mothos to have one method which accepts an argument of some input stream.
Then you can unit test that method using a mocked stream.
Then you chan check your varioius string parsers logic.
What do you think about that?

Reply:
It's not a bad idea at all. But considering this has languished for a year with no activity and was closed with no explanation, I don't believe it is worth my time. Apparently either this project is dead, or the people who could merge this don't consider this addition valuable, so I'm not going to waste my efforts further.